bundles/org.openhab.ui/web/src/components/app.vue

```diff
@@ -74,8 +74,8 @@
           </li>
 
           <f7-list-item link="/settings/addons/" :title="$t('sidebar.addOnStore')" panel-close :animate="false"
-                        :class="{ currentsection: currentUrl.indexOf('/settings/addons/') >= 0 }">
-            <f7-icon slot="media" ios="f7:bag" aurora="f7:bag" md="material:shopping_bag" color="gray" />
+                        :class="{ currentsection: currentUrl === '/settings/addons/' }">
```

Comment:
I think this will work to solve #2049 but introduce a new issue:
- http://localhost:8080/settings/addons/binding-astro will not highlight the Add-on Store section although it is part of it
- http://localhost:8080/settings/addons/binding-astro/config should still highlight Settings

Now that's a further argument that we should reorganize the routes so that add-on store pages are all under `/addons` and the settings page are under `/settings`.
The add-on settings pages (including persistence) should still be under `/settings/addons/:addonId/config` but the "store" pages (to browse and install) should be under `/addons`.

Reply:
> Now that's a further argument that we should reorganize the routes so that add-on store pages are all under /addons and the settings page are under /settings.

Yeah, I already put this on my todo list when you commented on #1997. 
I will take care of that once I’m back home.